.eslintignore

```diff
@@ -0,0 +1,11 @@
+# Ignore files not managed by ourselves
+client/app/resources/*
+client/app/test/*
+client/app/components/modal/*
+**/bower_components/*
+
+# Ignore modules
```

Comment:
No reason of doing this because in `Gruntfile.js` we specify: `target: assets.src.js` (https://github.com/ovh-ux/ovh-manager-web/blob/master/Assets.js#L5-L32)

Reply:
Pay attention to `insert_final_newline` (Editorconfig) :)